docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -592,18 +592,37 @@ How to **investigate**:
   - Invalid result block:
     - **How to detect**: Search compactor logs for `invalid result block`.
     - **What it means**: The compactor successfully validated the source blocks. But the validation of the result block after the compaction did not succeed. The result block was not uploaded and the compaction job will be retried.
-    - Out-of-order chunks
-      - **How to detect**: Search compactor logs for `invalid result block` and `out-of-order chunks`.
-      - This is caused by a bug in the ingester - see [mimir#1537](https://github.com/grafana/mimir/issues/1537). Ingesters upload blocks where the MinT and MaxT of some chunks don't match the first and last samples in the chunk. When the faulty chunks' MinT and MaxT overlap with other chunks, the compactor merges the chunks. Because one chunk's MinT and MaxT are incorrect the merge may be performed incorrectly, leading to OoO samples.
-      - **How to mitigate**: Mark the faulty blocks to avoid compacting them in the future:
-        - Find all affected compaction groups in the compactor logs. You will find them as `invalid result block /data/compact/<compaction_group>/<result_block>`.
-        - For each failed compaction job
-          - Pick one result block (doesn't matter which)
-          - Find source blocks for the compaction job: search for `msg="compact blocks"` and a mention of the result block ID.
-          - Mark the source blocks for no compaction (in this example the object storage backend is GCS):
-            ```
-            ./tools/markblocks/markblocks -backend gcs -gcs.bucket-name <bucket> -mark no-compact -tenant <tenant-id> -details "Leading to out-of-order chunks when compacting with other blocks" <block-1> <block-2>...
-            ```
+  - Out-of-order chunks
```

Comment:
Are you sure this section should no longer be beneath the _Invalid result block_ bullet point? I'm wondering if it's intentional that this item is nested beneath _Invalid result block_.

Reply:
no strong opinion form me, but I'd say that this feels like part of the "invalid result block" bullet point. I think it's clear both ways though.

Reply:
The multiple layers of bullets (6 layers!) made me think it was a mistake and I found it easier to read in this edited form - but if it's the intention I'm happy to undo the change, whatever people think is clearer :)